### ISSUE TITLE
Указывать имя формы, к которой относится ошибка в сообщении.

### ISSUE THREAD

**Issue by DmitryShvaika** [1C-Company/1c-edt-issues#70]
### Описание проблемы

В списке ошибок к формам объекта, который отображается в заголовке не понятно, к какой форме относится ошибка

![image](https://user-images.githubusercontent.com/54451084/140760242-960a9dbe-4209-4488-b540-6e703528d781.png)

### Описание решения проблемы

Предлагаю после слова в модуле формы писать имя формы, к которой относится эта ошибка.

**Comment by marmyshev** [comment]
По этому предложению - опишите, что мешает использоваться функциольность панели "Ошибки конфигурации"? Всё что предлагается - уже там реализовано, хотелось бы понять что мешает пользоваться этим?

**Comment by DmitryShvaika** [comment]
> По этому предложению - опишите, что мешает использоваться функциольность панели "Ошибки конфигурации"? Всё что предлагается - уже там реализовано, хотелось бы понять что мешает пользоваться этим?

На мой взгляд сообщение об ошибке было бы лучше, если вместо
_Для событий "ПриИзменении" элемента формы "Дата" назначен обработчик ДатаПриИзменении, но метод с таким именем отсутствует в модуле формы_
 написать так:
_Для событий "ПриИзменении" элемента "Дата" формы "ИмяФормы" назначен обработчик ДатаПриИзменении, но метод с таким именем отсутствует в модуле формы_
и сразу бы стало понятно - в какой форме ошибка

**Comment by RedMammoth** [comment]
Все же есть, зачем дублировать?
![image](https://user-images.githubusercontent.com/64589646/152340343-9a840483-552e-4b1b-9728-db8a2fe63cc9.png)

**Comment by DmitryShvaika** [comment]
> Все же есть, зачем дублировать? ![image](https://user-images.githubusercontent.com/64589646/152340343-9a840483-552e-4b1b-9728-db8a2fe63cc9.png)
> 
> Как мне кажется, чем больше в сообщении ошибки уникальных данных, тем сложнее потом анализировать список (т.к. ухудшается селективность). Захочется сгруппировать - нужно будет регуляркой вытаскивать из сообщения уникальные данные. А так нужно собрать статистику по сообщению ошибки - схлопнул по сообщению, нужна статистика по объектам - схлопнул по объекту, нужна статистика в разрезе и того и того - группируешь по двум полям и т.д.

Но если дописать всего одно имя формы в сообщении, то и в панель не понадобится лезть.... Мне кажется будет удобней - если прочитал и все сразу ясно

**Comment by RedMammoth** [comment]
> в панель не понадобиться лезть..
> если прочитал и все сразу ясно

А в каком окне прочитал, если не в панеле ошибок? я просто всегда читаю или в панеле ошибок (и там видно какой объект), или непосредственно в редакторе формы/модуля тут вопроса "где" вообще не возникает

**Comment by marmyshev** [comment]
Есть специальная панель, в которой сделано куча классной функциональности по работе с ошибками.

Ещё раз спрошу: есть ли какие-то проблемы с тем чтобы ей пользоваться?

@DmitryShvaika вы, вместо этого предлагает сделать из элемента «Декорация» в редакторе метаданного какую-то альтернативную систему с адресацией. Зачем это делать если уже есть удобный инструмент!? Или он чем-то неудобный?

 На мой вопрос «зачем» отвечаете выше «как надо сделать» в другом месте. В совершенно не предназначенном для этого месте!

**Comment by DmitryShvaika** [comment]
> Есть специальная панель, в которой сделано куча классной функциональности по работе с ошибками.
> 
> Ещё раз спрошу: есть ли какие-то проблемы с тем чтобы ей пользоваться?
> 
> @DmitryShvaika вы, вместо этого предлагает сделать из элемента «Декорация» в редакторе метаданного какую-то альтернативную систему с адресацией. Зачем это делать если уже есть удобный инструмент!? Или он чем-то неудобный?
> 
> На мой вопрос «зачем» отвечаете выше «как надо сделать» в другом месте. В совершенно не предназначенном для этого месте!
Нет, с панелью все ок... давайте закрою запрос - т.к. не ошибка вызывает противоречивые желания - при отображении ошибки в форме имя формы будет выглядеть действительно глупо.
